spark3-extensions/src/main/scala/org/apache/spark/sql/catalyst/optimizer/RewriteMergeInto.scala

```diff
@@ -266,5 +271,15 @@ case class RewriteMergeInto(spark: SparkSession) extends Rule[LogicalPlan] with
       }
     }
   }
+
+  private implicit class TableHelper(table: Table) {
```

Comment:
nit: I wonder whether it would make sense to return Iceberg `Table` here, instead of `SparkTable`.

Reply:
This broke the other table helper, so I removed it.